API.md

```diff
@@ -147,7 +147,9 @@ batch :: { k: v } -> ([a] -> Promise [b]) -> a -> Promise b
 
 | Option | Type | Default | Description |
 | ------ | ---- | ------- | ----------- |
+| `inputKey` | `a -> String` | | generates input key to match results with inputs |
 | `limit` | `Number` | `Infinity` | max length of each batch |
+| `outputKey` | `a -> String` | | generates output key to match results with inputs |
```

Comment:
Minor... but maybe `input/outputKeyFn` or `getInput/OutputKey`? `The `inputKey` noun implies a string in my head.